auth/templates/passwordReset.html

```diff
@@ -0,0 +1,5 @@
+<h2>Hello {name}. </h2>
+<p>Do you want to define a new passwd?</p>
+<p>follow the link to proceed: <a href="host/passwd/resetlink/{link}">LINK</p>
```

Comment:
We should probably be taking the host as an configuration (environment or file) variable and replacing it here, since hostname (or addressing more broadly) depends on the environment where dojot is running.